test/metabase/models/model_index_test.clj

```diff
@@ -53,16 +51,16 @@
                                                          :name "model index test"
                                                          :visualization_settings {}
                                                          :display "table"))
-                model       (t2/select-one :model/Card :id (:id response))
+                model       (t2/select-one :model/Card :id (u/the-id response))
```

Comment:
Was running into random test failures, by using `u/the-id` things fail if there's no ID in the response which means we don't run around trying to fetch Card `WHERE id IS NULL` because the API endpoint `response` was an error message and then having something break 20 frames later